hw05/Personel Database/BMICalculatorDB/BMICalculatorDB/Form1.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Data.SqlClient;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace BMICalculatorDB
+{
+    public partial class Form1 : Form
+    {
+        public Form1()
+        {
+            InitializeComponent();
+        }
+        SqlConnection con;
+        SqlCommand cmd;
+        private void Form1_Load(object sender, EventArgs e)
+        {
+        //    //string ConnString = "Data Source = localhost, Initial Catalog = EnhancedBMI, Integrated Security = true";
+        //    //con = new SqlConnection(ConnString);
+        //    //cmd = new SqlCommand();
+        }
+
+        private void buttonCalculate_Click(object sender, EventArgs e)
+        {
+            decimal weight = numericWeight.Value;
+            decimal height = numericHeight.Value;
+
+            if (weight == 0 || height == 0 || comboBoxGender.Text == null)
+            {
+                MessageBox.Show("Please enter a value into every box.");
+                return;
+            }
+            decimal bmi = (weight * 703) / (height * height);
+            textBoxBMI.Text = bmi.ToString();

```

Comment:
I think we should format this string so that it shows up to 2 or 3 decimal points.